tests/AdapterTest.php

```diff
@@ -34,7 +34,7 @@ protected function getEnvironmentSetUp($app)
                 'Token'           => null,
                 'Expiration'      => "@{$expire}",
             ];
-            return promise_for(new Response(200, [], stream_for(json_encode($f))));
+            return promise_for(new Response(200, [], \GuzzleHttp\Psr7\Utils::streamFor(json_encode($f))));
```

Comment:
This line fixes this build problem: 
![image](https://user-images.githubusercontent.com/55773428/160330093-7d3c761b-adf4-457b-842b-50e2d1428d34.png)
Thanks, Guzzle 2.2